modules/parent-join/src/yamlRestTest/resources/rest-api-spec/test/20_parent_join.yml

```diff
@@ -3,53 +3,30 @@ setup:
       indices.create:
         index: test
         body:
+          settings:
+            number_of_shards: 2 #Two shards proves that we route properly
           mappings:
             properties:
               join_field: { "type": "join", "relations": { "parent": "child", "child": "grand_child" } }
               id: { "type": "keyword" }
 
   - do:
-      index:
```

Comment:
I was borrowing this test setup and wanted to clean it up a bit while I was here. `bulk` is ever so slightly quicker for this sort of thing. And it takes up a lot fewer lines!